server.json

```diff
@@ -3,5 +3,8 @@
 		"./common",
 		"./node",
 		"./language/es2018"
-	]
+	],
+	"rules": {
+		"es-x/no-hashbang": "off"
```

Comment:
Yeah, but for some reason I can't remember it didn't work until I did it here too. Possibly a config load order issue?

Reply:
Indeed, this "off" instruction counter acts rules loaded by `/language/es2018`. The node.json file doesn't load this rule, so there is no reason for it to turn it off, either.

Maybe it's useful to keep it set in both, to ease customisations where downstream users use `common + node + es<different>`  instead of `server`. However, that would not work for the same reason it doesn't work here, namely that `es<different>` would override it. To avoid confusion, I'll remove the entry in node.json for now.